src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -19,6 +19,25 @@
 
 (def standard-line (str "1px solid " (:line-default colors)))
 
+(def header-row-style-light
+  {:fontWeight nil
+   :color (:text-light colors) :backgroundColor nil})
+
+(def table-row-style-light
+  {:borderTop standard-line
+   :padding "0.2rem 0"})
+
+(def table-cell-plank-left
+  {:borderRadius "8px 0 0 8px"
+   :margin "4px 0" :marginRight -20
+   :padding "0.6rem" :paddingRight "calc(0.6rem + 20px)"
+   :backgroundColor (:background-light colors)})
+
+(def table-cell-plank-right
+  {:borderRadius "0 8px 8px 0"
+   :margin "4px 0" :padding "0.6rem"
+   :backgroundColor (:background-light colors)})
+
```

Comment:
I agree with Putnam here. Please create a story to factor out table styles before merging.